packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -57,6 +57,7 @@ Requires: %{?scl_prefix}rubygem(deface) >= 1.0.2
 Requires: %{?scl_prefix}rubygem(deface) < 2.0.0
 Requires: %{?scl_prefix}rubygem(angular-rails-templates) >= 1.0.2
 Requires: %{?scl_prefix}rubygem(angular-rails-templates) < 1.1
+Requires: %{?scl_prefix}rubygem(fx) < 1.0
```

Comment:
This is not preset in the gemspec so the next automated update will remove this: https://github.com/theforeman/foreman-packaging/pull/5062